02-peer-protocol.md

```diff
@@ -559,6 +560,8 @@ A sending node:
         (witness program versions 1 through 16)
 
 A receiving node:
+  - if it have received multiple `shutdown`:
+    - should ignore the others after the first one.
```

Comment:
nit:
```suggestion
  - if it receives multiple `shutdown` messages:
    - SHOULD ignore all except the first one.
```

Reply:
Thanks! maybe before opening a PR a self-review was needed! Sorry about that